src/std/ide/swank/api.ss

```diff
@@ -0,0 +1,23 @@
+(import :std/misc/string)
+(export #t)
+(def current-slime-client (make-parameter #f))
+(def current-slime-writer (make-parameter #f))
+(def current-slime-package (make-parameter #f))
+(def current-slime-thread (make-parameter #f))
+(def current-slime-id (make-parameter #f))
+(def current-swank-exit-emacs-rex (make-parameter #f))
+(def (swank-machine:version)
+  (with ([cpu os kern] ##os-system-type-saved)
+    (if (eq? os 'apple)
+      (string-trim-eol
+       (cdr (##shell-command "sysctl -n machdep.cpu.brand_string" #t)))
```

Comment:
please use the canonical name -- `shell-command`, it is completely unnecessary top use the unsafe `##shell-command` here.